library/patient.inc

```diff
@@ -318,9 +318,16 @@ function getProviderInfo($providerID = "%", $providers_only = true, $facility =
     $param2 = "";
```

Comment:
a huge amount of prior code was reverted in this script. Recommend getting most recent script (note a bug was just fixed on this script about an hour ago) and then making your changes on that.